bay-services/graph-cve-sync.yaml

```diff
@@ -10,7 +10,7 @@ services:
       SYNC_MODE: diff #diff for differential sync and full for whole sync
       CRON_SCHEDULE: "0 */6 * * *"
       SNYK_DELTA_FEED_MODE: false
-      SNYK_INGESTION_FORCE_RUN: true
+      SNYK_INGESTION_FORCE_RUN: false
```

Comment:
what is this flag for? i thought we wanted to switch of delta feed for a day.

Reply:
Will do it tomorrow once the python is done.. today its doing for node

Reply:
force run is to run snyk feed forcefully.. else it runs once  a day.